### PR TITLE
fix(sql): fix parsing of explain command with parenthesized queries

### PR DESCRIPTION
**Pull request by bziobrowski**
Fixes parsing of explain command with parenthesized top-level query such as : 
```sql
explain 
(
  (select last(timestamp) as x, last(price) as btcusd 
  from trades 
  where symbol = 'BTC-USD' 
  and timestamp > dateadd('m', -30, now())) timestamp(x)
)
```

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 5 / 5 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|5|5|100.00%|

**Review by ideoma — COMMENTED** [review]
This works

```java
       assertPlan("select 1)",
                "VirtualRecord\n" +
                        "  functions: [1]\n" +
                        "    long_sequence count: 1\n");
```

but this does not 

```java
        assertPlan("select 1))",
                "VirtualRecord\n" +
                        "  functions: [1]\n" +
                        "    long_sequence count: 1\n");
```

Not a show stopper but strange

File: core/src/main/java/io/questdb/griffin/SqlParser.java [hunk 1]
```diff
@@ -1011,7 +1011,9 @@ private ExecutionModel parseExplain(GenericLexer lexer, SqlExecutionContext exec
         return parseSelect(lexer);
     }
 
-    private int parseExplainOptions(GenericLexer lexer) throws SqlException {
+    private int parseExplainOptions(GenericLexer lexer, CharSequence prevTok) throws SqlException {
+        int parenPosition = lexer.getPosition();
```
Comment (bluestreak01) [inline core/src/main/java/io/questdb/griffin/SqlParser.java:1015]: `parent`

**Comment by bziobrowski** [comment]
@ideoma It's not linked to explain or this pr but I fixed it in #6e336a56 .

File: core/src/main/java/io/questdb/griffin/SqlParser.java (hunk 1 above)
Comment (bziobrowski) [reply core/src/main/java/io/questdb/griffin/SqlParser.java]: Actually it's the position of parenthesis after explain . Fixed name in 6e336a56 .

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 22 / 24 (91.67%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|22|24|91.67%|

**Review by bluestreak01 — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 22 / 24 (91.67%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/griffin/SqlParser.java|22|24|91.67%|

**Pull request merged**